hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/DeletedBlockLogImpl.java

```diff
@@ -191,7 +200,7 @@ public int resetCount(List<Long> txIDs) throws IOException, TimeoutException {
     lock.lock();
     try {
       if (txIDs == null || txIDs.isEmpty()) {
-        txIDs = getFailedTransactions().stream()
+        txIDs = getFailedTransactions(-1).stream()
```

Comment:
I would avoid magic numbers, you can define some constants for these which make it obvious what is intended Example: `LIST_ALL_FAILED_TRANSACTIONS`